src/Pim/Bundle/UIBundle/Resources/public/less/base/general.less

```diff
@@ -44,6 +44,7 @@ body {
   margin: 0;
   padding: 0;
   overflow: hidden;
+  letter-spacing: -0.2px;
```

Comment:
I think not, I never heard issues with that (and letter spacing I set is very little)